scripts/csi-doctor.sh

```diff
@@ -82,8 +63,21 @@ debug_app_pod() {
   app_pod_uid=$(${kbctl} -n $namespace get po $app -o jsonpath='{.metadata.uid}')
   for pvc_name in $PVC_NAMES
   do
+    pv_name=$(${kbctl} -n ${namespace} get pvc $pvc_name -o jsonpath='{.spec.volumeName}' 2>/dev/null)
+
+    if [ "$pv_name" == "" ]; then
+      echo "Ignore $pvc_name due to empty volumeName"
+      continue
+    fi
+
+    provisioner=$(${kbctl} get pv ${pv_name} -o 'go-template={{index .metadata.annotations "pv.kubernetes.io/provisioned-by"}}' 2>/dev/null)
```

Comment:
If pv is created by users(static provision), its value is `pv.kubernetes.io/bound-by-controller: "yes"`. 

I recommend to use `pv.csi.driver == "csi.juicefs.com"` instead of annotation.

Reply:
good point, fixed